front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -154,6 +179,37 @@ export class CreateF3XStep1Component implements OnInit, OnDestroy {
   public goBack() {
     this.router.navigateByUrl('reports');
   }
+
+  public save() {
+    this.formSubmitted = true;
+
+    if (this.form.invalid) {
+      return;
+    }
+
+    this.form.patchValue({
+      coverage_from_date: this.convertDateToFecFormat(this.form.get('coverage_from_date')?.value),
+      coverage_through_date: this.convertDateToFecFormat(this.form.get('coverage_through_date')?.value),
+      date_of_election: this.convertDateToFecFormat(this.form.get('date_of_election')?.value),
+    });
+    const summary: F3xSummary = F3xSummary.fromJSON(this.validateService.getFormValues(this.form));
+    this.f3xSummaryService.create(summary, this.formProperties).subscribe(() => {
+      this.router.navigateByUrl('reports');
+      this.messageService.add({
+        severity: 'success',
+        summary: 'Successful',
+        detail: 'Contact Updated',
+        life: 3000,
+      });
+    });
+  }
+
+  public convertDateToFecFormat(date: Date | null) {
```

Comment:
I'm thinking this function may be needed in other components as well. How about moving it to a static method in a new shared/utils/date.utils.ts class?